client/blocks/site/index.jsx

```diff
@@ -151,7 +151,10 @@ class Site extends Component {
 		// We show public coming soon badge only when the site is not private.
 		// Check for `! site.is_private` to ensure two Coming Soon badges don't appear while we introduce public coming soon.
 		const shouldShowPublicComingSoonSiteBadge =
-			! site.is_private && this.props.site.is_coming_soon && ! isAtomicAndEditingToolkitDeactivated;
+			! site.is_private &&
+			this.props.site.is_coming_soon &&
+			! isAtomicAndEditingToolkitDeactivated &&
+			! isMigrationTrialSite( site );
```

Comment:
Should we also apply same behavior for Woo Trial site as well?

Reply:
I've tested it and figured out that when you select a site under the Woo trial plan, you will be redirected to the `wp-admin`, and this chunk of logic will never be executed. That's why I only handed migration trial case.